types/errors/handle.go

```diff
@@ -0,0 +1,12 @@
+package errors
+
+import "fmt"
+
+// Panic panics on error
+// Should be only used with interface methods, which require return error, but the
+// error is always nil
+func Panic(err error) {
```

Comment:
If we keep Panic (which I prefer not to), could we rename to e.g. `AssertNoError`? Panic seems that it will always panic.

Reply:
I can rename.

Reply:
To avoid name clash `errors.AssertNoError`, how about `errors.AssertNil`?

Reply:
I renamed to `errors.AssertNil`.